templates/named.conf.erb

```diff
@@ -21,6 +21,9 @@ options {
 <%-   end -%>
 	};
 <%- end -%>
+<%- if @forward != '' -%>
```

Comment:
To be clear here, "<%- if @variable and @variable != '' -%>" will be necessary to handle the possibility of undef per the [Puppet 4.1+ documentation](https://docs.puppet.com/puppet/4.8/reference/lang_data_boolean.html):

"If a non-boolean value is used where a boolean is required:
* The undef value is converted to boolean false.
* All other values are converted to boolean true.

Notably, this means the string values "" (zero-length string) and "false" both resolve to true."